tutorials/katacoda/1-serving-intro/step2.md

```diff
@@ -0,0 +1,26 @@
+## Deploy and autoscale application
+
+We are going to deploy the [Hello world sample web application](https://knative.dev/docs/serving/samples/hello-world/helloworld-go/). This basic web application reads in an env variable TARGET and prints `Hello ${TARGET}!`. If TARGET is not specified, it will use `World` as the TARGET.
+
+We will now deploy the application by specifying the image location and the `TARGET` env variable.
+
+Knative defines a `service.serving.knative.dev` CRD to control the lifecycle of the application (not to be confused with kubernetes service). We will use the `kn` cli to create the Knative service:
+
+```
+kn service create demo --image gcr.io/knative-samples/helloworld-go --env TARGET=blue
```

Comment:
```suggestion
kn service create demo --image gcr.io/knative-samples/helloworld-go --env TARGET=blue --autoscale-window 15s
```

what do you think of specify a 15 seconds, this way the user doesn't have to seat there waiting for 60 seconds